database/src/tests/builder.rs

```diff
@@ -116,13 +117,19 @@ pub struct JobBuilder {
     benchmark_set: u32,
     conclusion: BenchmarkJobConclusion,
     kind: BenchmarkJobKind,
+    is_optional: bool,
 }
 
 impl JobBuilder {
     pub fn profile(mut self, profile: Profile) -> Self {
         self.profile = profile;
         self
     }
+
+    pub fn is_optional(mut self, is_optional: bool) -> Self {
```

Comment:
```suggestion
    pub fn make_optional(mut self) -> Self {
```
Otherwise the name was confusing, it was a setter that begins with `is_`.

Reply:
:+1: 35a3aebd028e738a01ad6e267d7af1ced556811d